src/main/resources/templates/signup2.html

```diff
@@ -30,6 +30,11 @@ <h3 class="dashhead loginpanel" th:text="#{signup.header}">Registration</h3>
 			                            <li>&nbsp;</li>
 			                        </ul>
 
+									<p class="account-change-fail" th:if="${signupError}"><i class="fa fa-exclamation-circle"></i> <span th:text="${signupError}"></span></p>
+									<p class="account-change-fail" th:if="${#fields.hasErrors('email')}"><i class="fa fa-exclamation-circle"></i> <span th:errors="*{email}"></span></p>
+									<p class="account-change-fail" th:if="${#fields.hasErrors('password')}"><i class="fa fa-exclamation-circle"></i> <span th:errors="*{password}"></span></p>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)